### ISSUE TITLE
BSAs Not Added To INI

### ISSUE THREAD

**Issue by ghost** [Yacoby/Wrye-Mash#25]
I have map 2 of Tamriel Rebuilt and it has a .bsa that needs to be added to the Morrowind.ini file. Currently, Mash doesn't add the entry to the .ini file. It would be nice to have it do this when the associated .esm is enabled.

**Comment by Yacoby** [comment]
Related to #3

**Comment by ghost** [comment]
Sorry, should have searched first.

**Comment by Yacoby** [comment]
No, its fine. They are different (although related) issues. I just flagged that up so that anyone looking at either of those two issues would know to take a look at the other one.
